packages/amplify-e2e-core/src/categories/auth.ts

```diff
@@ -368,6 +368,23 @@ export function updateAuthSignInSignOutUrl(cwd: string, settings: any): Promise<
   });
 }
 
+export function updateAuthToRemoveFederation(cwd: string, settings: any): Promise<void> {
```

Comment:
nit: exclude settings parameter if not used.

Reply:
I will get this in a follow up PR